### ISSUE TITLE
Cannot access menu bar menu's

### ISSUE THREAD

**Issue by jamy015** [Swordfish90/cool-retro-term#261]
Hi, I just started using cool-retro-term a while ago, but unfortunately there seems to be a bug. I can't access the menu's in the menu bar, when I click one of the menu's the menu title is highlighted showing that the menu is selected, but the actual menu doesn't appear.

See the following screenshots, one when I haven't clicked the menu, and one when I have (notice that the menu title is highlighted with a white background, but the menu hasn't shown up):
![one](https://cloud.githubusercontent.com/assets/3188721/10334298/aaa0f86a-6ce9-11e5-91b9-f52b59e8de0f.png)
![two](https://cloud.githubusercontent.com/assets/3188721/10334303/ae9b0640-6ce9-11e5-9804-aa0f9bd3aa35.png)

I'm running 64-bit Debian 8.2 (jessie) with GNOME 3.14.1.

-Jamy

**Comment by thezerobit** [comment]
I can confirm this is also/still a problem on 64-bit Debian jessie version 8.4 in XFCE.

**Comment by LurkAshFlake** [comment]
Gentoo here same's problem

**Comment by Moter8** [comment]
+1 on Arch-Linux & Plasma5 Desktop

**Comment by adamleblanc501** [comment]
Has anyone found a solution to this yet? Because I am having this problem as well.

**Comment by tguless** [comment]
Also the case on Linux Mint Mate

**Comment by nicoandresr** [comment]
+1 on Arch with Awesome WM

**Comment by cmontcalm** [comment]
Was this issue ever resolved? I am also having this problem on Debian Jessie running KDE (in a VM through Virtual Box). Another thing that I have noticed is that when I right click on the terminal, I do not get the right click menu.

**Comment by Locario** [comment]
I have same problem here under openSUSE Tumbleweed, i have installed from repo and source, same problem, menus are invisible.

Please, give us some support, i love cool-retro-term!

**Comment by hevanaa** [comment]
In a quite clean Fedora 25 installation with Gnome desktop, I can't see the content of menus either. But in another Fedora 24 PC (that has been heavily used with lots of additional packages installed), the menus are visible. Is some component missing?

**Comment by Locario** [comment]
Hi, openSUSE fresh install with my graphics card driver installed from nvidia i can see menus, same os in a virtual machine cannot see menus

Enviado desde mi smartphone Sony Xperia™


---- Johan Heikkilä escribió ----


In a quite clean Fedora 25 installation with Gnome desktop, I can't see the content of menus either. But in another Fedora 24 PC (that has been heavily used with lots of additional packages installed), the menus are visible. Is some component missing?

**Comment by hevanaa** [comment]
That's correct. Menus don't work in a virtual machine, whereas it works in a physical PC with Intel graphics.

**Comment by art3mis9** [comment]
Hi,

Using this on Xubuntu 16.04 (LTS) and have no menu visibility. Has there been a fix for this?

**Comment by ysflrtby** [comment]
I meet the same problem on Debian 8.7 with gnome 3.14+3.

**Comment by hevanaa** [comment]
I think this has to do with accelerated graphics. My VM is not accelerated.

**Comment by ex-trim** [comment]
Gentoo with plasma5 desktop, nvidia 7600GS(driver version is 304.131).

Same problem

**Comment by Crossfire0mega** [comment]
I am also having the same issue on Ubuntu 16.04.1 LTS except that when i click on a menu item the outline of the dropdown appears but is transparent. 
![screenshot from 2017-01-24 17-54-50](https://cloud.githubusercontent.com/assets/12384940/22270595/4e799b82-e25e-11e6-9148-bab2cca8fb5f.png)

**Comment by solardiz** [comment]
With Fedora 23 in a VM, the drop-down menus occupy their correct area but are of their background color with no visible text in them. A workaround is to run:

```
Xnest :1 &
DISPLAY=:1 cool-retro-term
```

to adjust the terminal settings to one's liking, then use the terminal emulator normally (without `Xnest`, but also without visible menus again). Alternatively, the SQLite database may be accessed directly:

```
sqlite3 ~/.local/share/cool-retro-term/QML/OfflineStorage/Databases/*.sqlite
```

then `.schema`, `select * from settings;`, etc.

**Comment by AzurLighting** [comment]
Same here, im running a VM with Opensuse tumbleweed and menus aren't showing

**Comment by Bulo-north** [comment]
I have the same problem.
I use FreeBSD 11.1 with Mate in VirtualBox.
I've installed by adding  the package.

**Comment by duncan-bayne** [comment]
Same problem here - FreeBSD 11.1 on Lenovo X220 w/ Intel drivers installed.

**Comment by duncan-bayne** [comment]
I get the following messages when I run cool-retro-term:

```
libGL error: failed to open drm device: Permission denied
libGL error: failed to load driver: i965
default KB_LAYOUT_DIR:  "/usr/local/lib/qt5/qml/QMLTermWidget/kb-layouts"
QObject::connect: No such slot Konsole::TerminalDisplay_QML_140::close()
loadAllColorSchemes
Session::run() - program: "/usr/local/bin/zsh"
Session::run() - arguments: ("")
started!
QSqlDatabase: QSQLITE driver not loaded
QSqlDatabase: available drivers: 
QSqlQuery::prepare: database not open
qrc:/Storage.qml:35: Error: Driver not loaded Driver not loaded
Both point size and pixel size set. Using pixel size.
Undecodable sequence: \001b(hex)[?2004h
Undecodable sequence: \001b(hex)[?2004l
QSqlQuery::prepare: database not open
qrc:/Storage.qml:35: Error: Driver not loaded Driver not loaded
```

Will investigate.  I think this may be a permissions problem (at least in my case).

**Comment by duncan-bayne** [comment]
On my system, it was a permissions problem.  Adding my user to the `video` group in `/etc/group` and rebooting fixed the problem.

**Comment by workgena** [comment]
Have the same problem with Debian Gnome 3. adding me to `video` group does not help.

**Comment by kilrah** [comment]
Same issue on Win10 WSL, can both add PPA and install prebuilt binaries or compile myself but same happens in both cases.

**Comment by duncan-bayne** [comment]
@workgena I suspect group membership required on GNU/Linux is different to FreeBSD, i.e. that it's _not_ `video`, but I'm afraid I don't know what it might be.

**Comment by CatXMoonclaw** [comment]
Same issue, Ubuntu 18.04 LTS

**Comment by maxlefou** [comment]
Same problem, Manjaro 18 under XFCE. Are people actually working on a fix? Because i'm surprised this is still unresolved after 2 years.

**Comment by mrillusi0n** [comment]
![image](https://user-images.githubusercontent.com/37063502/74097074-c8a3a200-4b2d-11ea-9f37-81c0d1abb6aa.png)
This is how bad the menu looks on macOS.

**Comment by inkfeathers** [comment]
+1 on virtualized Mint 19.3. I enabled accelerated graphics for the VM and it solved the issue.
